app/assets/javascripts/discourse/tests/acceptance/admin-badges-award-test.js

```diff
@@ -0,0 +1,37 @@
+import {
+  acceptance,
+  exists,
+  query,
+} from "discourse/tests/helpers/qunit-helpers";
+import { click, visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import I18n from "I18n";
+
+acceptance("Admin - Badges - Mass Award", function (needs) {
+  needs.user();
+  test("when the badge can be granted multiple times", async function (assert) {
+    await visit("/admin/badges/award/new");
+    await click(
+      query(
+        '.admin-badge-list-item span[data-badge-name="Both image and icon"]'
+      ).parentElement
```

Comment:
Something doesn't feel right here. Why do we have to target the parent element? Can we add a class to the element so that we can click it directly?